docs/reference/commandline/node_ps.md

```diff
@@ -1,18 +1,18 @@
 <!--[metadata]>
 +++
-title = "node tasks"
-description = "The node tasks command description and usage"
+title = "node ps"
```

Comment:
no, first example should be good; although it's not released yet, by now links can exist to the old location; the `aliases` adds a redirect-page for the old URL